src/app/service/search-click/search-click.service.ts

```diff
@@ -0,0 +1,14 @@
+import { EventEmitter, Injectable, Output } from '@angular/core';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class SearchClickService {
+  public emitter: EventEmitter<boolean> = new EventEmitter();
```

Comment:
Please, rename 'emitter' to a more informative name

Reply:
fixed